src/tools/qt-gui/src/treeviewmodel.cpp

```diff
@@ -191,9 +204,8 @@ void TreeViewModel::importConfiguration(const QString& name, const QString& form
 	{
 		result = merger.mergeKeySet(MergeTask (BaseMergeKeys (base, root), OurMergeKeys (base, root), TheirMergeKeys (importedKeys, root), root));
 	}
-	catch(...)//TODO
+	catch(...)//TODO: Which exceptions are possible?
 	{
-		qDebug() << "Could not merge keysets";
 		emit showMessage(tr("Error"), tr("Could not merge keys."),"");
```

Comment:
The example shows how you get information about the conflicts. That should be present in the dialog. (its enough to have it just as text)